Core/Core/AppEnvironment/AppEnvironment.swift

```diff
@@ -55,12 +55,12 @@ open class AppEnvironment {
 
     public static let shared = AppEnvironment()
 
-    public func subscribe<U>(_ useCase: U, _ callback: @escaping Store<U>.EventHandler) -> Store<U> where U: UseCase {
-        return Store(env: self, useCase: useCase, eventHandler: callback)
+    public func subscribe<U>(_ useCase: U, context: NSManagedObjectContext? = nil, _ callback: @escaping Store<U>.EventHandler) -> Store<U> where U: UseCase {
+        return Store(env: self, useCase: useCase, context: context, eventHandler: callback)
     }
 
-    public func subscribe<Model>(scope: Scope, _ callback: @escaping Store<LocalUseCase<Model>>.EventHandler) -> Store<LocalUseCase<Model>> {
+    public func subscribe<Model>(scope: Scope, context: NSManagedObjectContext? = nil, _ callback: @escaping Store<LocalUseCase<Model>>.EventHandler) -> Store<LocalUseCase<Model>> {
         let useCase = LocalUseCase<Model>(scope: scope)
-        return subscribe(useCase, callback)
+        return subscribe(useCase, context: context, callback)
```

Comment:
Did we need Store frcs on a background thread?

Reply:
Nope. Thanks.